packages/babel-plugin-proposal-do-expressions/test/fixtures/do-expression-with-early-exit/call-expression/input.js

```diff
@@ -0,0 +1,9 @@
+function main() {
```

Comment:
Can you add some execution tests for the `do-expression-with-early-exit` suite? For an example, you can search `exec.js` within the plugin test suites.